rdf-star-cg-spec.html

```diff
@@ -411,7 +411,7 @@ <h2>Grammar</h2>
       <h2>Parsing</h2>
       <p>In contrast to [[N-TRIPLES]], N-Triples* allows recursion on the <a href="#grammar-production-nt-subject">`subject`</a> and <a href="#grammar-production-nt-object">`object`</a> productions.</p>
 
-    <p>An N-Triples* document defines an <a>RDF* graph</a> composed of a set of <a>RDF* triples</a>. The <a href="N-TRIPLES#grammar-production-triple">`triple`</a> production produces an <a>RDF* triple</a> defined by the terms constructed for <a href="#grammar-production-nt-subject">`subject`</a>, <a data-cite="N-TRIPLES#grammar-production-predicate">`predicate`</a> and <a href="#grammar-production-nt-object">`object`</a>.</p>
+    <p>An N-Triples* document defines an <a>RDF* graph</a> composed of a set of <a>RDF* triples</a>. The <a href="N-TRIPLES#grammar-production-triple">`triple`</a> production produces an <a>RDF* triple</a> composed of a <a href="#grammar-production-nt-subject">`subject`</a>, <a data-cite="N-TRIPLES#grammar-production-predicate">`predicate`</a> and <a href="#grammar-production-nt-object">`object`</a>.</p>
```

Comment:
```suggestion
    <p>An N-Triples* document defines an <a>RDF* graph</a> composed of a set of <a>RDF* triples</a>. The <a href="N-TRIPLES#grammar-production-triple">`triple`</a> production produces an <a>RDF* triple</a> composed of the terms constructed for <a href="#grammar-production-nt-subject">`subject`</a>, <a data-cite="N-TRIPLES#grammar-production-predicate">`predicate`</a> and <a href="#grammar-production-nt-object">`object`</a>.</p>
```

I agree that "defined by" was maybe unclear, but "terms constructed for" is important. `subject`, `predicate` and `object` in this sentence refer to the production rules in the grammar, not the general roles that terms can play.

Reply:
Despite my usual spec-militancy, have the impression that when talking about a grammar, "a subject, predicate and object" is understood to be talking about parsed terms, and that "the terms constructed for" might make people think there's some other, more meta process going on.

The only way to resolve this is to kidnap 20 typical sociologists and do A/B testing on them.

Reply:
I think @ericprud's simplification is clear enough.